src/views/view_sycl.hpp

```diff
@@ -160,7 +160,7 @@ struct VectorView<
     return eval(ndItem.get_global_id(0));
   }
 
-  SYCL_BLAS_INLINE scalar_t eval(cl::sycl::nd_item<1> ndItem) const {
+  SYCL_BLAS_INLINE const scalar_t eval(cl::sycl::nd_item<1> ndItem) const {
```

Comment:
Why is this const?

Reply:
I think this was supposed to be addressed in #174 but it seems to have just been missed.

Reply:
It was indeed fixed in the other branch, but I didn't catch it when merging `master` into this branch because there was no conflict for this one (this branch had the changes of the other before the change). Thanks for catching that